4001-5000/LOC4359Maaqud.xml

```diff
@@ -42,40 +42,38 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: place</change>
-         <change who="DN" when="2023-12-01">Expanded the record, included coordinates, added information on the library</change>
+         <change who="DN" when="2023-12-01">Expanded the record, included coordinates</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
             <place type="church">
-               <placeName xml:id="n1" xml:lang="gez">መዓቁዲ፡ ዮሐንስ፡ ወልደ፡ ነጐድጓድ፡</placeName>
-               <placeName corresp="#n1" xml:lang="en">Maʿāquddi Yoḥannǝs Walda Nagʷadgʷād</placeName>
-               <placeName xml:id="n2" xml:lang="gez">ዮሐንስ፡ መዓቁዲ፡</placeName>
-               <placeName corresp="#n2" xml:lang="en">Yoḥannǝs Maʿāquddi</placeName>
-               <placeName xml:id="n3" xml:lang="gez">ዮሐንስ፡ ዓዲ፡ አይኑ፡</placeName>               
-               <placeName corresp="#n3" xml:lang="en">Yoḥannǝs ʿĀddi ʿĀynu</placeName>
-               
+               <placeName xml:id="n1" xml:lang="gez">መዓቁዲ፡</placeName>
+               <placeName corresp="#n1" xml:lang="en">Maʿāquddi </placeName>
+               <placeName xml:id="n2" xml:lang="gez">ዮሐንስ፡</placeName>
+                              
                <country ref="LOC3010Ethiop"></country>
                <region notBefore="1991" type="RegionalState" ref="LOC5932Tegray">The regional State of Tǝgrāy, Mǝbrāq Tǝgrāy zone, 
                   waradā Ṣǝgeraddā, tabiyā Māy Kʷiḥā, qušat Maʿāquddi</region>               
+               <location>
+                  <geo>13.849072 39.447303</geo>
+                  <height unit="m">2405.0</height>                  
+               </location>     
+               <country ref="LOC3010Ethiop"></country>
+               <region notBefore="1991" type="RegionalState" ref="LOC5932Tegray">The regional State of Tǝgrāy</region>  
+               <settlement ref="LOC6653EastTegray" type="zone" when="2023-11-15">East Tǝgrāy zone</settlement>
+               <settlement ref="LOC3826Hawzen" type="warada" when="2023-11-15"> waradā  Ḥawzen</settlement>
+               <settlement when="2023-11-15" ref="LOC7408Maykwiha" type="tabiya">ṭabiyā Māy Kʷiḥā</settlement>
+               <settlement when="2023-11-15" ref="LOC4358Maaqud" type="qushat">qušat Maʿāquddi</settlement>
                <location>
                   <geo>13.849072 39.447303</geo>
                   <height unit="m">2405.0</height>
                </location>
-               <note>
```

Comment:
why deleted?

Reply:
I think the place name can stay

Reply:
The place name is Maaquddi

Reply:
Not quite I believe. There is a village Maaquddi which already has an ID LOC4358Maaqud (an area on the map). Then in the village there is the church of St John (a dot inside of the area), which as a place has the ID LOC4359Maaqud. In the church of Yohannes Maaquddi there is a collection of manuscripts which has the ID INS1005Maaqud. 
This LOC record is for the church (building).